style.css

```diff
@@ -0,0 +1,362 @@
+body {
+    margin: 0;
+    font-family: 'Roboto', sans-serif;
+    min-height: 100vh;
+    display: flex;
+    flex-direction: column;
+}
+
+.main-header {
+    background-color: white;
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 40px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    border-bottom: 1px solid #DDDDDD;
+    font-size: 16px;
+}
+
+.main-header__link {
+    margin-left: 12px;
+}
+
+.main-header__span {
+    margin-right: 12px;
+}
+
+.main-content {
+    margin-top: 40px;
+    flex: auto;
+}
+
+.main-content__h1 {
+    text-align: center;
+    margin: 0;
+    padding: 16px;
+    font-size: 24px;
+}
+
+.big {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-items: center;
+    margin: auto;
+}
+
+.group {
+    width: 296px;
+    height: 296px;
+    margin: 16px;
+    display: flex;
+    flex-direction: column;
+}
+
+.group__header {
+    border: 1px solid #DDDDDD;
+    font-size: 16px;
+    font-weight: 700;
+    padding: 12px;
+    box-sizing: border-box;
+    background-color: #EEEEEE;
+    height: 40px;
+}
+
+.group__main {
+    border: 1px solid #DDDDDD;
+    border-top: none;
+    flex: auto;
+    display: flex;
+    flex-direction: column;
+    padding: 12px;
+    box-sizing: border-box;
+    font-size: 12px;
+}
+
+.group__main__header {
```

Comment:
Please note that using "{0}__{1}__{2}" is not match BEM methodology

Reply:
I get it but I'll fix it maybe